### PR TITLE
Document default values in BuildResHPXML measure

### LINKED ISSUE #1252: Document default values in BuildResHPXML measure
Requested by a user. Can we document the default values that will be used when the user does not provide an input for a given BuildResHPXML argument? Ideally automated so that it stays in sync with actual defaulting code and documentation.

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Closes #1252.

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] ~Schematron validator (`EPvalidator.xml`) has been updated~
- [ ] ~Sample files have been added/updated (`openstudio tasks.rb update_hpxmls`)~
- [ ] ~Tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests/test*.rb` and/or `workflow/tests/test*.rb`)~
- [ ] ~Documentation has been updated~
- [ ] ~Changelog has been updated~
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected changes to simulation results of sample files

### CODE REVIEW

File: BuildResidentialHPXML/measure.xml [hunk 1]
```diff
@@ -92,7 +92,7 @@
     <argument>
       <name>simulation_control_timestep</name>
       <display_name>Simulation Control: Timestep</display_name>
-      <description>Value must be a divisor of 60. If not provided, the OS-HPXML default is used.</description>
+      <description>Value must be a divisor of 60. If not provided, the OS-HPXML default (see [HPXML Simulation Control](https://openstudio-hpxml.readthedocs.io/en/v1.7.0/workflow_inputs.html#hpxml-simulation-control)) is used.</description>
```
Comment (shorowit) [inline BuildResidentialHPXML/measure.xml:95]: Hmm. What will this look like in PAT?

Comment (joseph-robertson) [reply BuildResidentialHPXML/measure.xml:95]: I'd assume it will just be the whole string.

Comment (shorowit) [reply BuildResidentialHPXML/measure.xml:95]: Can you check? We might have to make this a url string and just turn it to a link for the README.

Comment (shorowit) [reply BuildResidentialHPXML/measure.xml:95]: Although even with a url string, I wonder if the user can select the text to copy/paste.

Comment (joseph-robertson) [reply BuildResidentialHPXML/measure.xml]: Confirmed in OSApp that with the "href" approach you can at least right-click and "Copy Link Location".

Comment (shorowit) [reply BuildResidentialHPXML/measure.xml]: I guess there isn't much we can do for PAT, it would need to be updated to better handle this.

**Review by shorowit — APPROVED** [review]
I didn't scrutinize every link, but the approach looks good. Thanks!

**Pull request merged**